client.go

```diff
@@ -14,6 +14,10 @@ import (
 	"time"
 )
 
+const (
+	replitDBURLFile = "/tmp/replitdb"
```

Comment:
```suggestion
	replitDBURLFile = "/repl/replitdb/url"
```

smol request, we shouldn't be using `/tmp`.

Reply:
> smol request, we shouldn't be using `/tmp`.

I agree, issue is that we needed some place that is writable by the Repl and `/tmp` was the one that made sense given this constraint. We can revise this in the future.

Reply:
ah for that we can do a quick conman change to `mkdir` that directory and grant write permissions to pid1. the problem is that once we release a new version of this package, we're committed to supporting it more or less for life, so it's a bit of a big deal to change it in the future ^^;; (almost impossible)